examples_tests/48.glTF/main.cpp

```diff
@@ -0,0 +1,75 @@
+#define _IRR_STATIC_LIB_
+#include <irrlicht.h>
+
+#include "../common/QToQuitEventReceiver.h"
+#include "../../ext/ScreenShot/ScreenShot.h"
+
+using namespace irr;
+using namespace core;
+
+int main()
+{
+	irr::SIrrlichtCreationParameters params;
+	params.Bits = 24; 
+	params.ZBufferBits = 24; 
+	params.DriverType = video::EDT_OPENGL; 
+	params.WindowSize = dimension2d<uint32_t>(1280, 720);
+	params.Fullscreen = false;
+	params.Vsync = true;
+	params.Doublebuffer = true;
+	params.Stencilbuffer = false;
+	auto device = createDeviceEx(params);
+
+	if (!device)
+		return 1;
+
+	device->getCursorControl()->setVisible(false);
+
+	QToQuitEventReceiver receiver;
+	device->setEventReceiver(&receiver);
+
+	auto* driver = device->getVideoDriver();
+    auto* sceneManager = device->getSceneManager();
+    auto* assetManager = device->getAssetManager();
+    auto* fileSystem = assetManager->getFileSystem();
+
+    asset::IAssetLoader::SAssetLoadParams loadingParams;
+    auto meshes_bundle = assetManager->getAsset("C:/Users/devsh/OneDrive/Pulpit/glTF-Sample-Models-master/2.0/BoxTextured/glTF/BoxTextured.gltf", loadingParams);
```

Comment:
you could make a submodule + repo with glTF test models in 3rdparty and only build ex 48 and init/fetch the glTF repo if option `NBL_BUILD_GLTF_TEST` is enabled

Reply:
ofc:)

Reply:
solved